transcript/src/main/java/com/transcriptanalyzer/transcript/User_Requests_Intents/Service/createBlocksVoiceflow.java

```diff
@@ -185,10 +147,101 @@ public static JsonObject create_button (String id, String intent){
     }
 
 
-        public static void main(String[] args) throws IOException {
-            add_block("molly.plunkett@mail.utoronto.ca",
-                    "TLICKMMR2022",
-                    "636ad5bf6ca8dfcaff607013",
-                    "Order pizza", "Order spicy pizza", "Get a refund");
+
+
+    /** The following functions are helper methods that create the non-customizable Json Objects
+     *  for the blocks we want to create.*/
+
```

Comment:
Yeah I think that was the idea, I think it is really helpful because then we don't get distracted by the object we're testing but more so what the test is actually supposed to be doing!

Reply:
Yes based on code smells, we had bloated code which is why in terms of better readability I moved the code to helper functions I left code that was important to the main function as is so when reading the code you can see what is being done to make the blocks/getToken